qcodes/instrument_drivers/tektronix/Keithley_6500.py

```diff
@@ -206,12 +214,12 @@ def _get_mode_param(self, parameter, parser):
         cmd = f'{mode}:{parameter}?'
         return parser(self.ask(cmd))
 
-    def _set_mode_param(self, parameter, value):
+    def _set_mode_param(self, parameter: str, value: Any) -> None:
```

Comment:
like in other drivers?
```suggestion
    def _set_mode_param(self, parameter: str, value: Union[str, float, bool]) -> None:
```